geometry_msgs/Vector3 origin
float32 pitch
morefusion_ycb_video/VoxelDimensions dims
float32[] values
uint32[] indices
int32 instance_id
uint32 class_id
